src/components/Input.jsx

```diff
@@ -97,4 +102,70 @@ class PasswordInputComponent extends Component {
   }
 }
 
+const PasswordStrength = props => {
+  const { password, className, ...rest } = props
+  const strength = passwordHelper.getStrength(password)
```

Comment:
Nit: If getStrength is expensive (since it uses regexes), a useMemo could be useful.

Reply:
This has been extracted to cozy-ui, in which the regex have been extracted from the helper function, so it's not recreated on every call.

Reply:
The matching of a regex can also be fairly expensive.